src/navigation/OccupancyGridClient.js

```diff
@@ -69,11 +69,13 @@ ROS3D.OccupancyGridClient.prototype.processMessage = function(message){
   // check for an old map
   if (this.currentGrid) {
     // check if it there is a tf client
-    if (this.currentGrid.tfClient) {
+    if (this.tfClient) {
```

Comment:
Would be great if we can add unit tests with each PR

Reply:
Would be great... but given our track record on testing(we have practically none) the cost for this PR submitter to accomplish testing significantly higher than their core contribution. We should accept this PR.